user-image/requirements.txt

```diff
@@ -46,3 +46,18 @@ pymc3==3.2
 daft==0.0.4
 corner==2.0.1
 #
+#
+# data-x DL
+#
+# TensorFlow
+tensorflow>=1.3.0
+# Image
+pillow
+scikit-image
```

Comment:
Can you put explicit versions on all these?

Reply:
if your install works and are the versions you want you can run `pip freeze` to see the versions